modules/core/src/layers/xviz-layer.js

```diff
@@ -30,81 +30,92 @@ import {XVIZObject} from '@xviz/parser';
 
 import deepExtend from 'lodash.merge';
 
+const XVIZ_PRIMITIVE_TYPES = {
+  circle: "circle",
+  image: "image",
+  point: "point",
+  polygon: "polygon",
+  polyline: "polyline",
+  stadium: "stadium",
+  text: "text"
+};
+
 const XVIZ_TO_LAYER_TYPE = {
   // V1
-  points2d: 'scatterplot',
```

Comment:
Lets not change the existing style unless there is some reason.  When in rome ...